packages/eslint-config-mocha/index.js

```diff
@@ -0,0 +1,7 @@
+
+module.exports = {
+    plugins: [
+        'mocha'
+    ],
+    extends: 'plugin:mocha/recommended'
```

Comment:
For consistency with the other rule sets I would wrap the extends with `[]`, so : `extends: ['plugin:mocha/recommended']`